src/main/java/com/jonathanbae/phototracker/PhotoController.java

```diff
@@ -0,0 +1,97 @@
+//create a controller with CRUD operations for: 
+//photos table with fields: 
+//    id (auto gen/increment)
+//	url
+//	description
+//	location
+//	
+//	
+//	
+//create new, get all, get by id, put (update by id), delete by id
+
+package com.jonathanbae.triptracker.controller;
+
+//import java.util.ArrayList;
+//import java.util.Arrays;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import com.jonathanbae.phototracker.Photo;
+import com.jonathanbae.triptracker.repository.PhotoRepository;
+
+import java.util.List;
+import java.util.Optional;
+
```

Comment:
format this file better, remove the unnecessary empty lines here (if using intellij, use command  + option + L